### PR TITLE
Update request module version from 2.83.0 to 2.87.0

### PR DESCRIPTION
**Pull request by dhavaldesai10**
### This fix has been created because latest version of request module does not have dependencies on hawk.

Details here
- connect to https://github.com/strongloop/loopback/issues/3889

### Checklist

- [ ] New tests added or existing tests modified to cover all changes
- [X] Code conforms with the [style
  guide](http://loopback.io/doc/en/contrib/style-guide.html)o new
- [X] Updated dependencies version

@bajtos @virkt25 : Could you please review my code?

### CODE REVIEW

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "Remoting",
     "REST"
   ],
-  "version": "3.11.0",
+  "version": "3.11.1",
```
Comment (virkt25) [inline package.json]: please revert this change, the version number is bumped automatically when we do a release.

**Comment by dhmlau** [comment]
@slnode test please

**Comment by dhmlau** [comment]
@dhavaldesai10 , as mentioned in the review comments, the change proposed in this PR is not needed.  With this, I'd like to close this PR.  Thanks for your interest in improving our framework!
